cypress/support/commands.ts

```diff
@@ -116,7 +116,7 @@ Cypress.Commands.add(
 Cypress.Commands.add(
   'createRun',
   ({ problemAlias, fixturePath, language }: RunOptions) => {
-    cy.visit('arena/problem/' + problemAlias);
+    cy.visit(`arena/problem/${problemAlias}/`);
```

Comment:
nomás faltaría [escaparlo](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent):

```suggestion
    cy.visit(`arena/problem/${encodeURIComponent(problemAlias)}/`);
```